### PR TITLE
Endpoint add tag

### PR DESCRIPTION
**Pull request by Kingdorian**
This adds an endpoint which can be used to add tags.

### CODE REVIEW

File: backend/src/main/java/com/benine/backend/http/AddTagHandler.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * Created by BeNine on 25-5-16.
+ */
+public class AddTagHandler extends RequestHandler {
+
+  @Override
+  public void handle(String s, Request request, HttpServletRequest httpServletRequest,
+                     HttpServletResponse httpServletResponse) throws IOException, ServletException {
+    String tagName = request.getParameter("name");
+    if (tagName != null) {
+      ServerController.getInstance().getPresetController().addTag(tagName);
```
Comment (jochem725) [inline backend/src/main/java/com/benine/backend/http/AddTagHandler.java]: Do we want to respond success even if the `tagname == null`?

**Comment by bvanwijk** [comment]
Looks good, I agree with merging.

**Comment by jochem725** [comment]
Seems fine, I've added a small comment, after that I think this can be merged.

**Comment by Kingdorian** [comment]
@bvanwijk It should be fixed now.

**Comment by bvanwijk** [comment]
Great fix, I think it can be merged now.

**Pull request merged**